public/index.html

```diff
@@ -185,6 +185,52 @@
 
     <hr/>
 
+		<uxtabarea>
+			<uxtablinkcontainer>
+				<uxtablink controls="firstTab" active="true">Tab One</uxtablink>
+				<uxtablink controls="secondTab">Tab Two</uxtablink>
+				<uxtablink controls="thirdTab">Tab Three</uxtablink>
```

Comment:
Something to discuss, should we just rely in order and maintain __id__ inernally instead of setting it explicitly ?